src/demo/pages/FocusZonePage/examples/FocusZone.Disabled.Example.tsx

```diff
@@ -2,16 +2,18 @@
 import * as React from 'react';
 /* tslint:enable:no-unused-variable */
 
-import { FocusZone, Button, TextField } from '../../../../index';
+import { FocusZone, Button, TextField, FocusZoneDirection } from '../../../../index';
 import './FocusZone.Disabled.Example.scss';
 
 export const FocusZoneDisabledExample = () => (
   <div className='ms-FocusZoneDisabledExample'>
     <div className='ms-Row'>
-      <FocusZone>
+      <FocusZone direction={ FocusZoneDirection.horizontal }>
         <span>Enabled FocusZone: </span>
         <Button>Button 1</Button>
         <Button>Button 2</Button>
+        <TextField value='FocusZone TextField' className='ms-FocusZoneDisabledExample-textField' />
```

Comment:
I did, works great. My change doesn't look at arrow direction, it just gets whether or not the next element "isForward" for determining which position the selection should be.